docs/builtins.rst

```diff
@@ -0,0 +1,217 @@
+================
+Builtins in L20n
+================
+
+L20n ships with a number of built-in functions that can be used from within
+localization messages.
+
+The list of built-ins is extensible and environments may want to add additional
+built-ins designated to aid localizers writing translations targeted
+for a given environment.
+
+
+How to use Built-ins
+====================
+
+Localizers can reference builtin in a message within a placeable and either
+display the return value, or use it in the selector expression to select
+the right value.
+
+Examples::
+
+
+  today-is = Today is { DATETIME($date, weekday=long) } 
+
+
+Default built-ins
+=================
+
+Default built-ins are very generic and should be applicable to any translation
+environment.
+
+``NUMBER``
+    Formats a number to a string in a given locale.
+    Accepts the same parameters as `Intl.NumberFormat`_ API.
```

Comment:
I know it's repetition but I think we should provide a quick summary of the available parameters in form of a table for each function.